_data/whatwedo.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Work based on https://github.com/mmistakes/minimal-mistakes/blob/master/_data/ui-text.yml
+#
+# Configuration: Add to _config.yml
+# locale: "en-US"  [YOUR PREFERRED LOCALE]
+#
+# How to use:
+# {{ site.data.ui-text[site.locale].<var_name> | default: 'text' }}
+
+# English (default)
+# -----------------
+en: &DEFAULT_EN
+
+  whatwedo:
+    section: whatwedo
+    title: "What we do"
+    text: "At CSC, we are passionate about improving healthcare in the NHS through the development of innovative new
```

Comment:
Not a fan of this copy. Can we just remove it?

Reply:
Sure.

Reply:
Removed